### ISSUE TITLE
Botania Relics cannot be made in Alchemical Matter Infuser due to No. of inputs

### ISSUE THREAD

**Issue by anabsolutesloth** [FTBTeam/FTB-Modpack-Issues#6340]
### Modpack

FTB Interactions Remastered

### Modpack version

1.9.0

### Launcher

Prism

### Has the pack been modified

No

### Describe the bug

Botania Relics require 3 unique items to make in the Alchemical matter Infuser, making them impossible to craft in the AMA, due to the structure only having 2 Item input slots in the form of Arcane Pedestals.

### Steps to reproduce

1. Search "Dice of Fate" in JEI and pull up uses.
2. Open Alchemical Matter Infuser tab.
3. Witness problem.
4. (Optional) Attempt (and fail) to put 3 items in 2 slots.

### Expected behaviour

Botania Relics to be craftable through the Alchemical Matter Infuser.

### Screenshots

![image](https://github.com/user-attachments/assets/2e3a55ce-a2a3-476e-96fc-01e4e88ad7b4)

### Additional information

The simplest way to resolve it would be to remove the Programmed Circuit from the recipes, as it doesn't matter, and can be easily stripped from the definitions for these recipes.
Another means would be to give AMA an additional input or the option of using Input Buses instead of Pedestals.

**Comment by ArtpokeBr** [comment]
Yeah, those recipes were overlooked, in update 1.11 you should be able to place configured circuits inside of the controller to properly configure the recipe =)
